StripePayments/StripePayments/Source/PaymentHandler/STPPaymentHandler.swift

```diff
@@ -1311,6 +1311,14 @@ public class STPPaymentHandler: NSObject {
             defer {
                 blockingDataTaskSemaphore.signal()
             }
+
+            // Check if the request failed due to attempting to redirect to an app universal link for Cash App
+            if let nsError = error as? NSError, let errorUrl = nsError.userInfo[NSURLErrorFailingURLStringErrorKey] as? String {
+                if errorUrl.contains("cashme://cash.app"), let universalLink = URL(string: errorUrl) {
+                    resultingUrl = universalLink
```

Comment:
Assuming we are okay with introducing a regression for users who don't have the app installed...

1. We should rename `universalLink` to something like `customSchemeAppURL`
2. Not ideal that we're having to hard-code this in here, but it does seem safer than checking for wildcards.